dashboard/app/models/peer_review.rb

```diff
@@ -142,9 +142,25 @@ def self.create_for_submission(user_level, level_source_id)
     end
   end
 
+  def self.get_review_completion_status(user, script)
+    if user &&
+        script.peer_reviews_to_complete.try(:>, 0) &&
+        Plc::EnrollmentUnitAssignment.exists?(user: user, plc_course_unit: script.plc_course_unit)
+      reviews_done = PeerReview.where(reviewer: user, script: script, status: [0, 1, 2]).size
```

Comment:
Will there be more statuses you don't want to include here, or it's just checking for non-nil? What about `status: PeerReview.statuses.values` or `.where.not(status: nil)`?

Or if you want to check specific ones and exclude escalated: `status: PeerReview.statuses.values_at(:accepted, :rejected)`?